pkg/components/strawberry_controller.go

```diff
@@ -221,6 +226,7 @@ func (c *StrawberryController) syncComponents(ctx context.Context) (err error) {
 			VolumeMounts: volumeMounts,
 		},
 	}
+	deployment.Spec.Template.Spec.Tolerations = c.microservice.getTolerations()
```

Comment:
Aren't they being set in `microservice.buildDeployment()` ?

Reply:
Yes, looks like you right, overengineered that a bit =)